locales/en/public.json

```diff
@@ -12,6 +12,56 @@
     "OPEN_SOURCE_LICENSE": "Open Source License",
     "VERSION": "version"
   },
+  "AutomatedAnalysisCard": {
+    "ARIA_LABELS": {
+      "DELETE_AA": "Delete automated analysis",
+      "REFRESH_AA": "Refresh automated analysis",
+      "TOOLBAR": "Automated Analysis Toolbar"
+    },
+    "CARD_DESCRIPTION": "Assess common application performance and configuration issues",
+    "CARD_DESCRIPTION_FULL": "Creates a recording and periodically evaluates various common problems in application configuration and performance. Results are displayed with scores from 0-100 with colour coding and in groups. This card should be unique on a dashboard",
+    "CARD_TITLE": "Automated Analysis",
+    "CHECKBOX_LABEL_SHOW_NA_SCORES": "Show N/A scores",
+    "CLEAR_FILTERS": "Clear all filters",
+    "CRITICAL_RESULTS_one": "{{count}} Critical Result",
+    "CRITICAL_RESULTS_other": "{{count}} Critical Results",
+    "ERROR_TEXT": "Cryostat was unable to generate an automated analysis report.",
+    "ERROR_TITLE": "Automated Analysis Error",
+    "GOOD_RESULTS": "No problems",
+    "NO_RESULTS": "No Results Found",
+    "NO_RESULTS_BODY": "No results match this filter criteria. Try removing filters, or resetting the severity score filter to 0 to show results",
+    "RESULT": "Name",
```

Comment:
Is this key used anywhere?

Reply:
Yes, when the Automated Analysis shows no warnings or critical results. https://github.com/cryostatio/cryostat-web/pull/862/files#diff-b48b9c2299bbdbf197673ec97eae7e57c2e00305d1e135c0da01777c150ec300R1015

Reply:
Oh if you are talking about the RESULT, then its here https://github.com/cryostatio/cryostat-web/pull/862/files#diff-b48b9c2299bbdbf197673ec97eae7e57c2e00305d1e135c0da01777c150ec300R718

Now that I look at it, I should rename it to NAME  and put it in public.js.